testeng/jobs/loadtestDriver.groovy

```diff
@@ -0,0 +1,112 @@
+ * factors).  Thus, this job relies on a human to manually abort it, or to set
+ * the MAX_RUN_TIME parameter.
+ *
+ * Instructions:
+ *
+ * 1. Ensure the "jenkins worker loadtest driver" AMI has been prepared (from
+ *    the jenkins_worker_loadtest.json packer script in edx/configuration).
+ * 2. On jenkins master, manually configure a new jenkins worker called
+ *    "loadtest-driver-worker" using the AMI from step 1.
+ * 3. On jenkins master, manually upload a credential file called
+ *    "loadtest_basic_auth_settings.yml" containing the current basic auth
+ *    credentials in the second YAML document.
+ *
+ */
+
+job('loadtest-driver') {
+
+    description('Start a new load test.')
+
+    // Abusing the team security feature to give all job control
+    // permissions to all edx employees.
+    authorization JENKINS_PUBLIC_TEAM_SECURITY.call(['edx'])
+
+    parameters {
+        stringParam('TARGET_URL', 'https://courses-loadtest.edx.org',
+                    'The loadtest target where requests are to be sent.')
+        stringParam('TEST_COMPONENT', 'lms',
+                    'The desired component to loadtest. See complete list here: ' +
+                    'https://github.com/edx/edx-load-tests/tree/master/loadtests')
+        stringParam('REMOTE_BRANCH', 'master',
+                    'Branch of the edx-load-tests repo to use.')
+        stringParam('NUM_CLIENTS', '',
+                    'This many locust clients will be hatched. This ' +
+                    'parameter must not be left empty.')
+        stringParam('HATCH_RATE', '',
+                    'Locust clients will be hatched at this rate ' +
+                    '(hatches/second). This parameter must not be left empty.')
+        stringParam('MAX_RUN_TIME', '',
+                    'After this amount of time the loadtest will ' +
+                    'automatically stop.  Its value is a floating point ' +
+                    'number with an optional suffix: \'s\' for seconds (the ' +
+                    'default), \'m\' for minutes, \'h\' for hours or \'d\' ' +
+                    'for days.  If left empty, the loadtest will never stop ' +
+                    'itself.')
+        fileParam('job_param_overrides.yml',
+                  'Override the default settings. This YAML file ' +
+                  'should use the standard edx-load-test settings format. ' +
+                  'DO NOT INCLUDE SENSITIVE SECRETS.')
+    }
+
+    /* It's okay to run multiple load tests at the same time.  We only require
+     * that they are on different workers, and that constraint is configured
+     * elsewhere.
+     */
+    concurrentBuild(true)
+
+    /* This worker label corresponds to an AMI built using
+     * util/packer/jenkins_worker_loadtest.json from edx/configuration.
+     */
+    label('loadtest-driver-worker')
+
+    scm {
+        git {
+            remote {
+                url('https://github.com/edx/edx-load-tests')
```

Comment:
Yep - moving to the new repo is definitely the goal, I was just checking that we weren't forgetting to make this configurable. 👍